manifests/csi/vsphere-csi-node-ds.yaml

```diff
@@ -0,0 +1,79 @@
+kind: DaemonSet
+apiVersion: apps/v1beta2
+metadata:
+  name: vsphere-csi-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: vsphere-csi-node
+  template:
+    metadata:
+      labels:
+        app: vsphere-csi-node
+        role: vsphere-csi
+    spec:
+      serviceAccount: vsphere-csi-node
+      hostNetwork: true
+      containers:
+        - name: driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.1
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins_registry/io.k8s.cloud-provider-vsphere.vsphere/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi/
+            - name: registration-dir
+              mountPath: /registration/
+        - name: vsphere-csi-node
+          image: gcr.io/cloud-provider-vsphere/vsphere-csi:v0.1.1
+          env:
+          - name: VSPHERE_KUBE_CONFIG
```

Comment:
I don't think this ENV var will be necessary for the Node Service.

Reply:
Yup, you are correct. Not needed. Just a copy and paste issue. WIll remove it.